corehq/motech/auth.py

```diff
@@ -123,8 +97,7 @@ def get_session(self, domain_name: str) -> Session:
         Returns an instance of requests.Session. Manages authentication
         tokens, if applicable.
         """
-        session = Session()
-        make_session_public_only(session, domain_name, src='sent_attempt')
+        session = get_public_only_session(domain_name, src='sent_attempt')
```

Comment:
If I change this line to `session = Session()` then tests in `./manage.py test corehq/motech/tests/test_requests.py` fail, so we can be sure the underlying functionality here is getting tested.